tests/netpyne-tests.js

```diff
@@ -152,8 +163,76 @@ function loadConsole(test, consoleButton, consoleContainer) {
     this.wait(1000)
   })
 }
-/*******************************************************************************
- *                                 popParams                                   *
+/******************************************************************************
+ *                                  appbar                                    *
+ ******************************************************************************/
+function testAppbar(test) {
```

Comment:
looks like the indentation is a bit of a mess here

Reply:
I am not using eslint here, so I dont get corrections. maybe we can tackle the issue in a separate PR .